src/Shared/PlatformSupport/AssetLoader.cs

```diff
@@ -242,6 +242,7 @@ private AssemblyDescriptor GetAssembly(string name)
                     throw new InvalidOperationException(
                         $"Assembly {name} needs to be referenced and explicitly loaded before loading resources");
 #else
+                    name = HttpUtility.UrlDecode(name);
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.uri.unescapedatastring?view=netframework-4.8

Reply:
Thanks for pointing that out, I did not know that exists